packages/node_modules/@node-red/runtime/locales/en-US/runtime.json

```diff
@@ -42,15 +42,16 @@
             "uninstall-failed-long": "Uninstall of module __name__ failed:",
             "uninstalled": "Uninstalled module: __name__"
         },
-        "deprecatedOption": "Use of __old__ is deprecated. Use __new__ instead",
+        "deprecatedOption": "Use of __old__ is DEPRECATED. Use __new__ instead",
         "unable-to-listen": "Unable to listen on __listenpath__",
         "port-in-use": "Error: port in use",
         "uncaught-exception": "Uncaught Exception:",
         "admin-ui-disabled": "Admin UI disabled",
         "now-running": "Server now running at __listenpath__",
         "failed-to-start": "Failed to start server:",
         "headless-mode": "Running in headless mode",
-        "httpadminauth-deprecated": "use of httpAdminAuth is deprecated. Use adminAuth instead",
+        "httpadminauth-deprecated": "Use of httpAdminAuth is DEPRECATED. Use adminAuth instead",
+        "httproot-deprecated": "Use of httpRoot is DEPRECATED. Use httpNodeRoot and httpAdminRoot instead",
```

Comment:
```suggestion
```

No need to add `httproot-deprecated` - we can use `deprecatedOption` as per my other feedback.

The existence of `httpadminauth-deprecated` predates `deprecatedOption` and could be updated to use `deprecatedOption` instead.